Userland/Applications/Spreadsheet/ImportDialog.cpp

```diff
@@ -175,13 +175,13 @@ void CSVImportDialogPage::update_preview()
     m_data_preview_table_view->update();
 }
 
-Result<NonnullRefPtrVector<Sheet>, DeprecatedString> ImportDialog::make_and_run_for(GUI::Window& parent, StringView mime, Core::File& file, Workbook& workbook)
+ErrorOr<NonnullRefPtrVector<Sheet>> ImportDialog::make_and_run_for(GUI::Window& parent, StringView mime, Core::File& file, Workbook& workbook)
```

Comment:
For this one, it looks like the code wanted to show additional error context than just "It failed". 

We can keep using a custom error type, in this case, DeprecatedString, for that. `ErrorOr<T, DeprecatedString>`.

In VideoPlayer/LibVideo, we called it `DecoderError` I believe? Don't think we need something as fancy as that though. 

It seems sad to lose all that context for a GUI App where it's intended for display.

Reply:
That's quite against the advantage of `ErrorOr`, that is being allocation-free.
But let's not lose the additional information until we found a better solution for this.